ank/doc/swdesign/plantuml/seq_get_agent.plantuml

```diff
@@ -0,0 +1,47 @@
+@startuml seq_get_agent
+!include ../../../../resources/doc/plantuml/clean.skin
+
+box "Ankaios Ank"
+    actor "user" as user
+    participant Main as main
+    participant "CliCommands" as cli_commands
+    participant "ServerConnection" as server_connection
+    participant "<<task>>\nCommunicationsClient" as communications_client
+end box
+
+box "Ankaios Server"
+    participant "Ankaios Server" as server
+end box
+
+-> main ++ : get agents
+
+== startup ==
+
+... initialization ...
+
+== command ==
+
+main -> cli_commands ++ : get agents
+
+cli_commands -> server_connection ++: request complete state
+server_connection ->> communications_client++: request complete state
+communications_client ->> server --++ : request complete state
+
+server_connection -> server_connection ++: wait for the response
+communications_client <<- server --++: complete state
+server_connection <<- communications_client --: complete state
+deactivate server_connection
+cli_commands <<- server_connection --: complete state
+
+cli_commands -> cli_commands ++: count workloads per agent\nin complete state
+cli_commands -> cli_commands ++: for each connected agent\nassign number of workload states\nof workloads it manages
+cli_commands -> cli_commands ++: format output text
+main <<-- cli_commands -- : output text
+deactivate cli_commands
+deactivate cli_commands
+deactivate cli_commands
+
+main -> main: print text
+user <- main -- : exit
```

Comment:
```suggestion
user <<-- main -- : exit
```

Reply:
Em in general it is meant as a destroy arrow but somehow the cross is not shown. In the `seq_cmd_overview.plantuml` however with the same syntax it is shown. I will check this.

Reply:
The destroy main is missing.

Reply:
I think just do a return and exit would be better than destroy. I will change it in all occurences.

Reply:
I have changed it for all get diagrams, but kept the destroy inside `seq_cmd_overview.plantuml` since in the comment next to the exit main part is mentioned what is done during upon exit.